userspace/libscap/scap_fds.c

```diff
@@ -603,7 +603,7 @@ int32_t scap_add_fd_to_proc_table(scap_t *handle, scap_threadinfo *tinfo, scap_f
 	}
 	else
 	{
-		handle->m_proc_callback(handle->m_proc_callback_context, tinfo->tid, tinfo, fdi, handle);
+		handle->m_proc_callback(handle->m_proc_callback_context, handle, tinfo->tid, tinfo, fdi, handle);
```

Comment:
I see the same argument `handle` passed twice on this function. Isn't redundant?

Reply:
ficed here a9672c56e4702e42f818ba9a02ced533a49d4136